python/pyspark/pandas/testing/utils.py

```diff
@@ -1,448 +0,0 @@
-#
```

Comment:
If we remove this file, we can also remove `__init__.py`?

Reply:
Makes sense!